test/config.conf

```
@@ -1,7 +1,11 @@
 [community-edition]
-hosts : 127.0.0.1:3000
+hosts:bob-cluster-a:3000
```

```suggestion
hosts : 127.0.0.1:3000
```
Changes for testing shouldn't be in production